src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+}).then(data => {
+  successHandler(data);
+}).catch(error => {
+  errorHandler(error);
+});
+
+new Promise((resolve, reject) => {
```

Comment:
```suggestion
new Promise((resolve) => {
```

Reply:
You can remove it if you are not using `reject` :)

Reply:
done